adapter/internal/adapter/adapter.go

```diff
@@ -196,6 +197,13 @@ func Run(conf *config.Config) {
 		})
 	}
 
+	// Start the metrics server
+	if conf.Adapter.Metrics.Enabled && strings.ToLower(conf.Adapter.Metrics.Type) == metrics.PrometheusMetricType {
+		logger.LoggerMgw.Info("Starting Prometheus Metrics Server ....")
+		go metrics.StartPrometheusMetricsServer(conf.Adapter.Metrics.Port, conf.Adapter.Metrics.CollectionInterval)
+
+	}
+
 	logger.LoggerMgw.Info("Starting adapter ....")
```

Comment:
Can we keep logs in this format ?

```suggestion
logger.LoggerMgw.Info("Starting adapter ....")
	
	// Start the metrics server
	if conf.Adapter.Metrics.Enabled && strings.ToLower(conf.Adapter.Metrics.Type) == metrics.PrometheusMetricType {
		logger.LoggerMgw.Info("Starting Prometheus Metrics Server ....")
		go metrics.StartPrometheusMetricsServer(conf.Adapter.Metrics.Port, conf.Adapter.Metrics.CollectionInterval)

	}


```